### DISCUSSION TITLE
Pending Ones --- @Announcement

### DISCUSSION THREAD

**Discussion by thegreywalker** [The-Tech-Firm/Web-Dev-Projects#11] [Announcements]
Ok guys @himangshuishere @TheArUnOdAyA  ... Welcome to the new Discussion group ... We will be discussing about subtle works left to do on this repo  and general changes we can make to look and make it better ...

Welcome all ...

# Task 1 ~
- [ ] Himangshu @himangshuishere Upload the The Tech Firm DP on the website also the Organisation DP .... ASAP

**Comment by ghost** [comment]
Ok working on it...

**Comment by ThEArUnOdAyA** [comment]
Ok got it.
@iamapurba2003

**Comment by thegreywalker** [comment]
Yah @himangshuishere @TheArUnOdAyA ... Carry on

**Comment by thegreywalker** [comment]
Guys @himangshuishere @ThEArUnOdAyA  is it done ? ... Are you done with editing the Photo @himangshuishere ?

**Reply by ghost** [reply]
@iamapurba2003 Working on it...

**Reply by thegreywalker** [reply]
Make it fast @himangshuishere
